bokehjs/src/coffee/models/glyphs/line.coffee

```diff
@@ -51,7 +51,7 @@ class LineView extends Glyph.View
       x: this.renderer.plot_view.canvas.vx_to_sx(geometry.vx)
       y: this.renderer.plot_view.canvas.vy_to_sy(geometry.vy)
     shortest = 9999
-    threshold = Math.max(2, @visuals.line.width.value() / 2)
+    threshold = Math.max(2, @visuals.line.line_width.value() / 2)
```

Comment:
@bryevdv I came across this, and assumed it slipped through during earlier refactoring (because everything is `line_xxx` now). Can you confirm this is supposed to be `line_width`?